RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,106 @@
+#ifndef RecoTracker_LSTCore_interface_alpaka_Constants_h
+#define RecoTracker_LSTCore_interface_alpaka_Constants_h
+
+#include "RecoTracker/LSTCore/interface/Constants.h"
+
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+#include <cuda_fp16.h>
+#endif
+
+namespace lst {
+
+  using namespace ALPAKA_ACCELERATOR_NAMESPACE;
```

Comment:
The `using` only applies to the scope of the `lst` namespace in this file (not any file that imports it), so shouldn't it be fine?

Reply:
My worry is that a type alias defined only in `ALPAKA_ACCELERATOR_NAMESPACE` (i.e. not in `alpaka_common`) becomes defined also in `lst`, can then quite easily be used in a function that otherwise looks like is independent of backends, but has e.g. `if constexpr (std::is_same_v<...>)`, that would lead to an ODR violation.

I'm not sure what you meant with 
> The `using` only applies to the scope of the `lst` namespace in this file (not any file that imports it)

It applies to all files that `#include` the `Constants.h`.

On a quick look, in the present version and in the `Constants.h` only the `Vec3D` and `Idx` type aliases seem to be used. These are independent of the backends. If those are the only use, how about `using namespace alpaka_common`?